### PR TITLE
1444453: Set default encoding for gui to UTF-8

### PR DESCRIPTION
**Pull request by kahowell**

### CODE REVIEW

File: bin/subscription-manager-gui [hunk 1]
```diff
@@ -22,6 +22,10 @@
 if __name__ != '__main__':
     raise ImportError("module cannot be imported")
 
+import sys
```
Comment (barnabycourt) [inline bin/subscription-manager-gui]: An explanatory comment before this block would be helpful for folks coming back later.

**Comment by awood** [comment]
My only request is that we add a comment explaining what's going on here.  Maybe a link to http://stackoverflow.com/a/29832646/6124862

**Comment by kahowell** [comment]
81ac204..724f8c5: comments added to `setdefaultencoding` usage in all bin scripts we use this in.

@awood @barnabycourt FYI.

**Comment by jirihnidek** [comment]
Hi @kahowell,
when you add comment or any string including any UTF-8 string, then it is necessary to add following string at the beginning of .py file:

```python
#!/usr/bin/python
# -*- coding: utf-8 -*-
```

Otherwise you will see following error and script will be terminated:

    [root@rhel01 subscription_manager]# ./bin/subscription-manager version
      File "./bin/subscription-manager", line 25
    SyntaxError: Non-ASCII character '\xd0' in file ./bin/subscription-manager on line 25, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details

> Note: when Python 3.x is used, then it is not necessary to add `# -*- coding: utf-8 -*-` at beginning of py file.

Please reopen this PR and merge it with suggested change to master.

**Pull request merged**